protostar/commands/test/__init__.py

```diff
@@ -1 +1,23 @@
+import sys
```

Comment:
I think this patch will affect the `build` command because Protostar imports all commands in `protostar_cli.py`. I'd move this code to `TestExecutionEnvironment::invoke_test_case`.

Reply:
But patching this in `TestExecutionEnvironment::invoke_test_case` won't be effective, as original hashing function will already be imported and won't be patched for most invokations in cairo-lang itself (and there are over 11000 calls)

Reply:
Moreover, I have just tried to check how much calls test execution command makes to hashing, and it is ~20 calls, so patching only this won't yield any gains :( 

In comparison, the way I'm doing it right now results in ~30% faster tests (and I have just measured - there are exactly 16900 calls to the function), so I think it actually should affect the build command in order to make the tests run faster..

Reply:
NVM. We would have to use lazy imports in each command.

Reply:
As discussed offline, such approach would still result in most of the calls being made to "original" hashing function, as other modules/classes import it indirectly, so a more "top-level" patch is desired.

@MaksymilianDemitraszek what do you think?

Reply:
I understand @kasperski95 concern, there is some risk connected to messing with cairo-lang code, but crypto-cpp is an offical starknet package. I think we can patch it everyhere. Eventually cairo-lang will support this AFAIK.

@THenry14 In this case can you please move this patch to global init as it affects everything? Sorry causing an extra effort :p

Reply:
No problem :+1: done